core/src/main/java/io/confluent/kafka/schemaregistry/exceptions/InvalidVersionException.java

```diff
@@ -37,4 +39,14 @@ public InvalidVersionException(Throwable cause) {
   public InvalidVersionException() {
     super();
   }
+
+  // Builder method rather than override string constructor
+  public InvalidVersionException withVersion(String version) {
```

Comment:
Generally, builder methods are static. Would it be just simpler to use a constructor here rather than a builder?

Reply:
I believe there are other usages of the constructor that takes a string and does not contain specifically the version string, which is why I did it this way. Plus, I wanted a getter for the version specially rather than the full exception message. 

Could be static, though. `InvalidVersionException.forVersion()`?